village/src/components/SmurfForm.js

```diff
@@ -15,19 +19,35 @@ class SmurfForm extends Component {
     // add code to create the smurf using the api
 
     this.setState({
-      name: '',
-      age: '',
-      height: ''
+      name: event.target.name.value,
+      age: event.target.age.value,
+      height: event.target.height.value
     });
+
+    axios.post('http://localhost:3333/smurfs', this.state )
+    .then(res => {
+      console.log(res.data);
+      this.setState( {
+        newSmurf : res.data[res.data.length-1]
+      })
+    })
+    .catch(err => {
+      console.log(err);
+    })
+    this.props.history.push('/smurfs')
+    window.location.reload();
```

Comment:
If you lift this method up to App and set the smurfs property there, there will be no need to reload